testdata/packages/mssql-0.9.0/docs/README.md

```diff
@@ -0,0 +1,211 @@
+Note: This integration is in beta!
+
+The MSSQL integration allows to monitor the SQL server and fetch logs. 
+
+![metrics-dashboard](../img/metrics-dashboard.jpg "Metrics Dashboard")
+
+
+# Compatibility
+
+This version is tested with SQL Server 2017.
+
+# Configuration
+
+TODO: Add docs on how to configure SQL server if any special configs needed.
+
+If haven't setup the agent yet, checkout the docs here. To manually configure the agent, use the following configuration.
+```
+inputs:
+- type: log
+  paths: "/var/opt/mssql/log/error*"
+  pipeline: mssql-access-pipeline
+
+- type: metric/mssql
+  metricsets: ["transaction_log", "performance"]
+  period: 10s
+  enabled: true
+  hosts: ["sqlserver://sa@localhost"]
+```
+
+
+  
+# Logs
+
+| Field  Name               | Description                                             |
+|----------------------------|---------------------------------------------------------|
+| mssql.log.origin           | Origin of the log event                                 |
+| message                    | Message from the access log                             |
+
+An example event looks as following:
+
+<details>
+    <summary>Click to expand</summary>
+        
+```
+{
+    "@timestamp": "2019-05-03T09:01:09.990Z",
+    "ecs.version": "1.0.0",
+    "event.dataset": "mssql.log",
+    "event.module": "mssql",
+    "event.timezone": "+00:00",
+    "fileset.name": "log",
+    "input.type": "log",
+    "log.flags": [
+        "multiline"
+    ],
+    "log.offset": 0,
+    "log.original": "2019-05-03 09:01:09.99 Server      Microsoft SQL Server 2017 (RTM-CU13) (KB4466404) - 14.0.3048.4 (X64)\n\tNov 30 2018 12:57:58\n\tCopyright (C) 2017 Microsoft Corporation\n\tDeveloper Edition (64-bit) on Linux (Ubuntu 16.04.5 LTS)",
+    "message": "Microsoft SQL Server 2017 (RTM-CU13) (KB4466404) - 14.0.3048.4 (X64)",
+    "mssql.log.origin": "Server",
+    "service.type": "mssql"
+}
+```
+</details>
+
+# Metrics
+
+## Performance metrics
+
+The first thing that any user will want to look at is, as you have probably guessed, performance. 
+It is very important is track the number of transactions, connected users, and cache hits, and all of this 
+is included in this metricset. These metrics are retrieved from Performance Counters in Microsoft SQL Server
+
+| Metric  Name               | Description                                             |
+|----------------------------|---------------------------------------------------------|
+| user_connections           | Total number of user connections                        |
+| logins_per_sec             | Total number of logins started per second               |
+| connections_reset_per_sec  | Total number of logins started from the connection pool |
+| transactions               | Total number of transactions                            |
+
+For an example event, click below.
+
+<details>
```

Comment:
@hbharding @jfsiii I expect us to have an option to not show all code together at once but let the user expand it if needed. Not sure if it's something special we need in markdown or if that can just work magically.